about.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about-style.css">
+
+    <title>About Comoputing </title>
+</head>
+
+<body>
+
+    <section class="Container">
+        <header>
+            <nav class="mobnav">
+                <div class="mobhead">
+                    <a href="#"> <img src="images/hamburger-icon.png" alt="hamburger" id="open-Button"></a>
+                </div>
+            </nav>
+            <!-- dropdown -->
+            <div class="snav" id="sNav">
+                <a href="#" class="c-btn" id="close-Button"><img src="./images/cross-button.png"
+                        alt="Cross"></a>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Join</a></li>
+                    <li><a href="#">Sponsor</a></li>
+                </ul>
+            <!-- nav for desktop -->
+
+            <nav class="topNav">
+                <ul>
+                    <li><a href="#"><img src="./images/facebook.svg" alt="facebook"></a></li>
+                    <li><a href="#"><img src="./images/twitter.svg" alt="twitter"></a></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+                    </nav>
+                    <nav class="dtopNav">
+                        <a href="index.html">
```

Comment:
In your desktop view for the about me page I noticed you are missing a the two navigation bars it would be nice to fix this issue.

This is your design:

 
![about page](https://user-images.githubusercontent.com/79506105/225371876-01f6f096-5c08-4911-9246-08d99e76b448.png)



Please follow this design:
![Screenshot 2023-03-15 at 5 57 27 PM](https://user-images.githubusercontent.com/79506105/225371987-7078a6ab-dfa3-41b4-b844-dd472fa3f923.png)


Then in your mobile view of your. about page please kindly add a background image like it is in the desktop screen.

This is your design:

![Screenshot 2023-03-15 at 5 59 45 PM](https://user-images.githubusercontent.com/79506105/225372982-226c4898-f4bc-458f-bc5c-c0a82d71eaf2.png)

secondly kindly fix the footer design of your about page in Both your desktop view and  mobile view .

These are your designs:

![abiut nav](https://user-images.githubusercontent.com/79506105/225373725-f68bcdfe-4d01-48d8-9f5a-8a692a42ac8a.png)

![Screenshot 2023-03-15 at 6 00 53 PM](https://user-images.githubusercontent.com/79506105/225373774-8dac9c3e-3272-49a4-a2a4-19d02bf5039a.png)


Please lets stick to this design for our footer 
![about nav 1](https://user-images.githubusercontent.com/79506105/225373945-83a32a47-500c-4b93-8205-69e194552ca6.png)
![
![Screenshot 2023-03-15 at 6 23 19 PM](https://user-images.githubusercontent.com/79506105/225374598-383e10a2-f294-4ac7-8382-7419dea6a6f5.png)

![Screenshot 2023-03-15 at 6 23 19 PM](https://user-images.githubusercontent.com/79506105/225374730-db1879c1-180f-441c-bdec-3b10e8363b5e.png)